build/common.sh

```diff
@@ -814,11 +814,11 @@ function kube::release::gcs::copy_release_artifacts() {
   fi
 
   # Now upload everything in release directory
-  gsutil -m "${gcs_options[@]+${gcs_options[@]}}" cp -r "${RELEASE_DIR}"/* "${gcs_destination}"
+  gsutil -q -m "${gcs_options[@]+${gcs_options[@]}}" cp -r "${RELEASE_DIR}"/* "${gcs_destination}"
 
   # Having the "template" scripts from the GCE cluster deploy hosted with the
   # release is useful for GKE.  Copy everything from that directory up also.
-  gsutil -m "${gcs_options[@]+${gcs_options[@]}}" cp \
```

Comment:
I did. The multithreading doesn't help for a single file. (I guess it doesn't hurt to leave it, though.)